VideoLocker/src/main/java/org/edx/mobile/util/WebViewUtil.java

```diff
@@ -0,0 +1,34 @@
+package org.edx.mobile.util;
+
+import android.content.Context;
+
+import org.edx.mobile.logger.Logger;
+
+import java.io.IOException;
+
+/**
+ * Common webview helper for any view that needs to use a webview.
+ */
+public class WebViewUtil {
+
+    /**
+     * Creates the intial StringBuffer used when an view uses a webview. Uses a common
+     * css file.
+     */
+    public static StringBuffer getIntialWebviewBuffer(Context context, Logger logger) {
+        StringBuffer buff = new StringBuffer();
```

Comment:
Will change. 

http://stackoverflow.com/questions/355089/stringbuilder-and-stringbuffer